.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  workflow_dispatch:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node Env
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: https://npm.pkg.github.com/
+      - name: Run npm install
+        run: npm ci
+        env:
+          GH_PACKAGES_NPM_TOKEN: ${{secrets.GH_PACKAGES_NPM_TOKEN}}
+      - name: Run npm test
+        run: npm test
+        env:
+          GH_PACKAGES_NPM_TOKEN: ${{secrets.GH_PACKAGES_NPM_TOKEN}}
+
+  publish-gpr:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: https://npm.pkg.github.com/
+      - name: Run npm install
+        run: npm ci
+        env:
+          GH_PACKAGES_NPM_TOKEN: ${{secrets.GH_PACKAGES_NPM_TOKEN}}
+      - run: npm run build
+        env:
+          GH_PACKAGES_NPM_TOKEN: ${{secrets.GH_PACKAGES_NPM_TOKEN}}
+      - run: npm publish
+        env:
+          GH_PACKAGES_NPM_TOKEN: ${{secrets.GH_PACKAGES_NPM_TOKEN}}
```

Comment:
Is this our private package registry or public?

Reply:
this is just the one we've used before. is switching this to a different token how you make it public? I did a bit of googling and apparently passing `npm publish --access public` should do the trick

Reply:
I haven't published a public NPM package before! .npmrc doesn't use this env var, so for a public package we don't actually need it.

Reply:
Ok, I removed all references to it. I thought maybe you'd need it to put something inside of `@compassion-technology`

Reply:
From looking things up it looks like I'll probably need to add it to `npmrc` so that I can publish this package to the org